### ISSUE TITLE
Photo Upload/Rotate Problems

### ISSUE THREAD

**Issue by queenmother305** [moxi9/phpfox#1402]
Several users photos are being uploaded sideways for some reason that needs to be addressed. Also when I attempted to correct the users photos by rotating them the rotation routine never completes. The wheel continuously spins. See attached.

![screenshot 15](https://cloud.githubusercontent.com/assets/19175383/15062416/d83302da-130d-11e6-83c8-2d53b0fefb64.png)

**Comment by Vivian-phpFox** [comment]
Can you reproduce this issue on demo site?

**Comment by queenmother305** [comment]
I checked the demo site but only someone with admin privelages has the option to rotate photos. I cannot check it because the rotate icon is not there for my user group.

**Comment by neil96** [comment]
Can't reproduce both on my localhost and demo.

**Comment by queenmother305** [comment]
try uploading photos from mobile phone. once uploaded return to your desktop and attempt to rotate them, specifically the ones that are sideways. i assure you there is some type of bug at work here.

**Comment by queenmother305** [comment]
my website is adult themed and high volume. may i post screenshots to demo the problem so it can be diagnosed and fixed? i don't want to offend anyone with the screenshots as they can be a bit xxx.

**Comment by Data66PF** [comment]
https://community.phpfox.com/forum/thread/49553/unable-to-upload-photos/ that issue with the sideways photos, also in the forum are old posts regarding this issue. At the time, it was related to server configuration and it timing out. I don't recall what setting needed to be changed and can't find the threads for it. I believe Fern had posted some guidelines. Perhaps someone else can find the other posts.

**Comment by Data66PF** [comment]
Regarding the demo, no adult photos can be loaded there.

**Comment by queenmother305** [comment]
no i wasnt gonna upload photos on your demo. i meant can i provide screenshots of my site for here to show the problem im having. my site has adult photos.

**Comment by Data66PF** [comment]
No please don't post the images here unless you actually blur out the adult content.

**Comment by Data66PF** [comment]
https://v4.phpfox.com/photo/3091/may-11-2016/refresh_1/

confirmed at demo. I added a new account. I uploaded a photo from an iphone for my profile. It is an iPhone 5c. 

I then used my Admin user and on my mac notebook, I went to that profile photo the other account had loaded and I try to rotate it. It never finishes and just spins and spins. https://v4.phpfox.com/photo/album/profile/7392/profile-7392/ It is now broken.
